File: Interpreter/Interpreter/Models/serialize/statementTypes/NameSpace.cs
using System.Collections.Generic;

namespace Interpreter.Models.serialize.statementTypes{
    public class NameSpace: IStatement
    {
        public bool isInline { set; get; }
        
        public string name { set; get; }

        public IList<IStatement> declarations = new List<IStatement>();

    }
}

File: src/main/kotlin/com/stannis/dataModel/statementTypes/NameSpace.kt
package com.stannis.dataModel.statementTypes

import com.stannis.dataModel.Statement

data class NameSpace(
    override val `$type`: String? = "NameSpace",
    var isInline: Boolean?,
    var name: String?,
    var declarations: ArrayList<Statement>?
) : Statement {
    fun addDeclaration(statement: Statement) {
        if(declarations == null) {
            declarations = ArrayList()
        }
        declarations!!.add(statement)
    }
}
